templates/azure/shared/_azure-logo-strip.html

```diff
@@ -1,41 +1,34 @@
-<section class="p-strip is-shallow u-no-padding">
+<section class="p-section">
   <div class="u-fixed-width">
+    <hr />
+    <h2 class="p-text--small-caps">Get Ubuntu Pro for Azure</h2>
     <div class="p-logo-section">
       <div class="p-logo-section__items u-no-margin--bottom">
-        <div class="p-logo-section__item"> 
-          {{ image (
-            url="https://assets.ubuntu.com/v1/d5a5c48f-canonical-logo.png",
-            alt="Canonical",
-            width="288",
-            height="288",
-            hi_def=True,
-            attrs={"class": "p-logo-section__logo"},
-            loading="auto"
-            ) | safe
+        <div class="p-logo-section__item">
+          {{ image(url="https://assets.ubuntu.com/v1/bd03aa14-canonical-logo.png",
```

Comment:
Did you run linting on this file? Image templates should look like so 
![image](https://github.com/user-attachments/assets/0f2ba41e-e9e3-4e56-b6c7-217ec983d93b)

Reply:
For some reason it simultaneously does not get those, will run once again